elyra/tests/pipeline/airflow/test_component_parser_airflow.py

```diff
@@ -52,7 +53,8 @@ def test_component_catalog_can_load_components_from_registries(component_cache_i
     assert len(components) > 0
 
 
-def test_modify_component_catalogs():
+@pytest.mark.usefixtures('teardown_test_catalog')
+def test_modify_component_catalogs(teardown_test_catalog):
```

Comment:
This seems redundant.  We usually only use the parameter form for "declaring" a fixture.